.github/workflows/broken-links.yml

```diff
@@ -0,0 +1,28 @@
+name: Broken-Links
+
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: "11 11 * * 0"   # once a week at 11:11 UTC on Sunday, avoid rush hours
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write # required for peter-evans/create-issue-from-file
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          fail: false
+          output: ./lychee/out.md
+      - name: Create Issue From File
+        if: steps.lychee.outputs.exit_code != 0
+        uses: peter-evans/create-issue-from-file@v5
+        with:
+          title: Link Checker Report
+          content-filepath: ./lychee/out.md
+          labels: report, automated issue
```

Comment:
Neither of these labels exist
How about we just use the "dead-link" label you made
I don't think we need a lot of labels on this repo tbh. We don't have a lot of issues to organise